packages/devextreme/testing/testcafe/runner.js

```diff
@@ -18,17 +18,8 @@ const changeTheme = async(themeName) => createTestCafe.ClientFunction(() => new
 
 const addShadowRootTree = async function() {
     await createTestCafe.ClientFunction(() => {
-        const root = document.querySelector('#parentContainer');
-        const childNodes = root.childNodes;
-
-        if(!root.shadowRoot) {
-            root.attachShadow({ mode: 'open' });
-        }
-
-        const shadowContainer = document.createElement('div');
-        shadowContainer.append.apply(shadowContainer, Array.from(childNodes));
-
-        root.shadowRoot.appendChild(shadowContainer);
+        // eslint-disable-next-line no-undef
+        window.addShadowRootTree();
```

Comment:
let's make `addShadowRootTree` clientFunction? Now it's js script, but if it will be clientFunction, it can be written in TS
You still can have separate file for it